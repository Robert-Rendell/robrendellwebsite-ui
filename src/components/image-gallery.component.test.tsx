import React from "react";
import { render } from "@testing-library/react";
import { ImageGallery } from "./image-gallery.component";

describe("ImageGallery Component", () => {
  test("should match snapshot", () => {
    const { asFragment } = render(<ImageGallery images={[]}/>);
    expect(asFragment()).toMatchSnapshot();
  });
});