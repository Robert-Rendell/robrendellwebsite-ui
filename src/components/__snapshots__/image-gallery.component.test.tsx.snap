// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ImageGallery Component should match snapshot 1`] = `
<DocumentFragment>
  <h2>
    Loading images from AWS S3...
  </h2>
  <img
    src="infinity-spinner.svg"
  />
</DocumentFragment>
`;
